### PR TITLE
Fix TestActivityHeartbeatTimeouts flakiness

### PR DESCRIPTION
**Pull request by alexshtin**
**What changed?**
Improved timeouts, logging, and use `atomic` to fix `TestActivityHeartbeatTimeouts` flakiness.

**Why?**
Test is flaky.

**How did you test it?**
Run it.

**Potential risks**
No risks.

### CODE REVIEW

File: common/persistence/cassandra/cassandraPersistenceTest.go [hunk 1]
```diff
@@ -135,25 +137,25 @@ func (s *TestCluster) CreateSession() {
...
 	if err != nil {
-		log.Fatal(`CreateSession`, err)
+		s.logger.Fatal("CreateSession", tag.Error(err))
 	}
 }
 
 // CreateDatabase from PersistenceTestCluster interface
 func (s *TestCluster) CreateDatabase() {
-	err := CreateCassandraKeyspace(s.session, s.DatabaseName(), 1, true)
+	err := CreateCassandraKeyspace(s.session, s.DatabaseName(), 1, false, s.logger)
```
Comment (alexshtin) [inline common/persistence/cassandra/cassandraPersistenceTest.go:146]: !

File: host/activity_test.go [hunk 2]
```diff
@@ -945,7 +946,7 @@ func (s *integrationSuite) TestActivityHeartbeatTimeouts() {
 						ScheduleToCloseTimeoutSeconds: 60,
 						ScheduleToStartTimeoutSeconds: 5,
 						StartToCloseTimeoutSeconds:    60,
-						HeartbeatTimeoutSeconds:       5,
+						HeartbeatTimeoutSeconds:       4,
```
Comment (alexshtin) [inline host/activity_test.go:949]: Turn timouts.

File: host/activity_test.go [hunk 3]
```diff
@@ -921,7 +922,7 @@ func (s *integrationSuite) TestActivityHeartbeatTimeouts() {
 
 	s.Logger.Info("StartWorkflowExecution", tag.WorkflowRunID(we.RunId))
 
-	workflowComplete := false
+	workflowComplete := &atomic.Bool{}
```
Comment (alexshtin) [inline host/activity_test.go:925]: Use atomic.

Comment (samarabbas) [reply host/activity_test.go:925]: I'm not sure how turning this into atomic helps.  This variable is never accessed concurrently.

File: common/persistence/cassandra/cassandraPersistenceTest.go (hunk 1 above)
Comment (markmark206) [reply common/persistence/cassandra/cassandraPersistenceTest.go:146]: just curious -- how does this make a difference? are you thinking test suits are dropping db from under each other (despite test suites seemingly generating a random db name for every run)?

File: common/persistence/cassandra/cassandraPersistenceTest.go [hunk 4]
```diff
@@ -171,7 +173,7 @@ func (s *TestCluster) LoadVisibilitySchema(fileNames []string, schemaDir string)
 	workflowSchemaDir := path.Join(schemaDir, "visibility")
 	err := loadCassandraSchema(workflowSchemaDir, fileNames, s.cluster.Hosts, s.cluster.Port, s.DatabaseName(), false, nil)
 	if err != nil && !strings.Contains(err.Error(), "AlreadyExists") {
-		log.Fatal(err)
+		s.logger.Fatal("loadCassandraSchema", tag.Error(err))
```
Comment (mastermanu) [inline common/persistence/cassandra/cassandraPersistenceTest.go:176]: [Nit] rename this one to "loadCassandraVisibilitySchema" just so it is clearer which schema failed to actually load

File: host/activity_test.go (hunk 3 above)
Comment (alexshtin) [reply host/activity_test.go:925]: My understanding is `wtHandler` and `atHandler` works in parallel. Am I wrong?

**Comment by alexshtin** [comment]
All above, didn't actually help but I found one more place to fix.

File: common/persistence/cassandra/cassandraPersistenceTest.go (hunk 1 above)
Comment (alexshtin) [reply common/persistence/cassandra/cassandraPersistenceTest.go:146]: Don't ask. But apparently it is not a root of the problem.

Comment (alexshtin) [reply common/persistence/cassandra/cassandraPersistenceTest.go:146]: I am reverting it back.

File: host/activity_test.go (hunk 3 above)
Comment (samarabbas) [reply host/activity_test.go:925]: They are only invoked when we poll for tasks, which is done sequentially in the test.

**Pull request merged**